source/draw/RasterMonochrome.ooc

```diff
@@ -76,41 +76,53 @@ RasterMonochrome: class extends RasterPacked {
 	_resizeNearestNeighbour: static func (source, result: This) {
 		resultBuffer := result buffer pointer
 		sourceBuffer := source buffer pointer
-		for (row in 0 .. result size height) {
-			sourceRow := (source size height * row) / result size height
-			for (column in 0 .. result size width) {
-				sourceColumn := (source size width * column) / result size width
-				resultBuffer[column + result stride * row] = sourceBuffer[sourceColumn + source stride * sourceRow]
+		resultHeight := result size height
+		resultWidth := result size width
+		sourceHeight := source size height
+		sourceWidth := source size width
+		resultStride := result stride
+		sourceStride := source stride
```

Comment:
And yes, I do think this reads more clearly. 6 short, confusingly similar looking lines is not good.